### PR TITLE
Add option to render Rothamsted model to example

### PR DESCRIPTION
**Pull request by alexdewar**
Hey Jamie,

This just adds support for loading the Rothamsted 3D model to the ant world example. You need the ``ROTHAMSTED_3D_MODEL_PATH`` environment variable set and then you run like so:
```
./ant_world_test --rothamsted
```

### CODE REVIEW

File: libantworld/common.h [hunk 1]
```diff
@@ -3,6 +3,9 @@
 // Third-party includes
 #include "../third_party/units.h"
 
+// OpenGL includes
+#include <GL/glew.h>
+
```
[the remark this answers is not in the document]
Comment (alexdewar) [reply libantworld/common.h]: Ah yes, very true! I've amended this and pushed.

**Comment by alexdewar** [comment]
Hmm is Jenkins not doing its thing? Seems to be taking a while...

**Comment by neworderofjamie** [comment]
Oddly the hook for some reason failed to err hook - it should be running now.

**Comment by alexdewar** [comment]
:+1:

**Pull request merged**